packages/web-components/src/components/link-list/link-list-item.ts

```diff
@@ -0,0 +1,68 @@
+/**
+ * @license
+ *
+ * Copyright IBM Corp. 2020
+ *
+ * This source code is licensed under the Apache-2.0 license found in the
+ * LICENSE file in the root directory of this source tree.
+ */
+import { html, customElement, LitElement } from 'lit-element';
+import ddsSettings from '@carbon/ibmdotcom-utilities/es/utilities/settings/settings';
+import styles from './link-list.scss';
+
+const { stablePrefix: ddsPrefix } = ddsSettings;
+
+/**
+ * Style schemes
+ */
+export enum STYLE_SCHEME {
+  /**
+   * Default style
+   */
+  DEFAULT = '',
+
+  /**
+   * Link List variant
+   */
+  LINK_LIST = 'link-list',
+}
+
+/**
+ * Link list item.
+ *
+ * @element dds-link-list-item
+ */
+@customElement(`${ddsPrefix}-link-list-item`)
+class DDSLinkListItem extends LitElement {
+  // eslint-disable-nesxt-line class-methods-use-this
+  protected render() {
+    return html`
+      <slot @slotchange=${this._handleSlotChange} />
+    `;
+  }
+
+  connectedCallback() {
+    super.connectedCallback();
+    if (!this.hasAttribute('role')) {
+      this.setAttribute('role', 'listitem');
+    }
+  }
+
+  /**
+   * handler, toggles link list style scheme on child components.
+   *
+   * @param e onSlotChange Event
+   * @private
+   */
+  // eslint-disable-next-line class-methods-use-this
+  private _handleSlotChange(e) {
+    const nodes = e.target.assignedNodes();
```

Comment:
```suggestion
  private _handleSlotChange(event: Event) {
    const nodes = (event.target as HTMLSlotElement).assignedNodes();
```

Reply:
@RaphaelAmadeu fyi we also created a yarn script for updating the snapshots: `yarn test:unit:updateSnapshot`